harvard-lts/fits
# LIBDRS-8701 Update to latest OTS; fix SCM element

**JIRA Ticket**: (link)

# What does this Pull Request do?
A brief description of what the intended result of the PR will be and/or what problem it solves.

# How should this be tested?
Using the file `docker/Dockerfile-test` see comments at top for how to create a Docker container and run test suite using this container.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n/a
- integration tests? n/a

# Interested parties
@awoods @cvicary
